triage/harness/validate.py

```
@@ -0,0 +1,155 @@
+
+
+def _check_no_empty_report(run_dir: Path) -> Dict[str, Any]:
+    """Schemas must reject ceremonial or empty reports."""
+    for jp in run_dir.glob("*.json"):
+        try:
+            data = json.loads(jp.read_text(encoding="utf-8"))
+            if isinstance(data, dict):
+                if len(data) <= 1:
+                    return {"name": "no_empty_report", "status": "FAIL", "message": f"{jp.name} is empty/ceremonial"}
+        except Exception:
+            continue
```

Ruff (0.15.21)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)